src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,40 @@
+.catalog {
+  display: grid;
+  grid-auto-columns: $card-width;
+  grid-auto-rows: $card-height;
+  grid-gap: 48px;
+  justify-items: start;
+  align-items: stretch;
+
+  &__card {
+    width: $card-width;
+    justify-self: stretch;
+    align-self: stretch;
```

you can use this to center text
```suggestion
    display: grid;
    place-items: center;
```